my-app/src/Error.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './index.css';
+
+
+interface errorProps {
+    errorId: number;
+    onSubmit: (e: React.ChangeEvent<HTMLFormElement>) => void;
+}
+
+export class Error extends React.Component<errorProps> {
+
+    render() {
+
+        let errorMessage: string;
+        if (this.props.errorId == 1) {
```

Comment:
```suggestion
        if (this.props.errorId === 1) {
```
基本javascriptでは `==` を使う場面はないと思って良いです